lib/commentOnClosedIssue.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+var Cesium = require('cesium');
+var Promise = require('bluebird');
+var rp = require('request-promise');
+
+var getUniqueMatch = require('./getUniqueMatch');
+
+var defined = Cesium.defined;
+
+/** Get comments -> regex search -> post comment
+ *
+ * @param {String} jsonResponse GitHub Response
+ * @param {Object} headers Request headers
+ * @return {Promise<http.IncomingMessage>} Response
+ */
+var commentOnClosedIssue = function(jsonResponse, headers) {
+    if (!defined(jsonResponse) || !defined(headers)) {
+        return Promise.reject('jsonResponse is undefined');
+    }
+    var commentsUrl = jsonResponse.issue.comments_url;
```

Comment:
I was going to suggest that we verify `issue` and `comments_url` both exist and throw an error otherwise, but it looks like `jsonResponse.issue.comments_url` is the only variable used from the `jsonReponse` object passed into this function.  Wouldn't it be better for this function to take just the `commentsUrl` string?